### ISSUE TITLE
Seeking clarification that SC 2.5.7 has no exception for web content used by the author but generated by the user agent

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1977]
There is an exception in [dragging movements](https://www.w3.org/TR/WCAG22/#dragging-movements) for interacting with the user agent - for example operating a scrollbar in the browser.  Does this exception apply to things such as a slider in a video player provided by the user agent where a single point activation is not available?     Perhaps all user agents today do support this - but I think it is important for us to be clear how the user agent exception might apply to default controls provided by the user agent but used by the author.

**Comment by alastc** [comment]
I'd describe that as a note which clarifies the scope of the guidelines in general, applied to this SC. I.e. none of the guidelines are aimed at user-agent functionality.

I think the scenarios in mind for that note were things like scrolling and swiping / gestures for mobile screen-readers.

For things like browser-provided video controls, that seems like a bit of a grey area. If you plonk in a `<video>` element without customisations, you are leaving the user-agent to define the behaviour of the slider. If you customise it or create your own, then you are defining the behaviour.

I think it would still be considered "web content", therefore in scope of the SC. Does anyone disagree?
(Disagree that it applies, separate from whether it should.)

**Comment by alastc** [comment]
I'm inclined to close this unless someone can provide an example where it's an issue?

**Comment by mbgower** [comment]
Draft response:
> Does this exception apply to things such as a slider in a video player provided by the user agent where a single point activation is not available?

As Alastair noted, this is something of a grey area, since the intention was not to force authors to overcome basic scrolling behaviour and other user agent default. A native video player is typically considered a user agent, so an argument could be made it meets the user agent wording. However, since no one seems able to provide an example of a user agent implementation of a stock control that fails dragging, it appears to be a non-issue.

**Comment by alastc** [comment]
Agreed today: https://www.w3.org/2021/12/03-ag-minutes.html#t05
